presets/models/inference/llama2-chat/inference-api.py

```diff
@@ -1,21 +1,21 @@
 # Copyright (c) Microsoft Corporation.
```

Comment:
Python just ordered the libraries alphabetically here upon saving - no changes to inference